nilearn/glm/first_level/design_matrix.py

```diff
@@ -406,6 +409,9 @@ def make_first_level_design_matrix(
 
     # step 1: events-related regressors
     if events is not None:
+        if not isinstance(events, (list, tuple)):
+            events = [events]
+        events = check_and_load_tables(events, "events")[0]
         # create the condition-related regressors
```

Comment:
If we accept list, and we only use the first entity, we could alternatively write this line as:

`check_and_load_tables(events[0], "events")[0]`

to avoid to load all files in the list and use only the first one.

Reply:
modified check_and_load_tables to cast to list if the input is not one, so that should solve the issue